### PR TITLE
Pictures on, alerts on posts if incomplete

### PR DESCRIPTION
**Pull request by noahnovickf**

### CODE REVIEW

File: src/containers/post/post.component.jsx [hunk 1]
```diff
@@ -46,6 +50,23 @@ const Post = ({ post, user, fetchLatestPosts }) => {
...
+        .getDownloadURL()
+        .then((img1Url) => {
+          setImg1(img1Url);
+        });
+      storage
+        .ref(`posts/${picID}-02`)
+        .getDownloadURL()
+        .then((img2Url) => {
+          setImg2(img2Url);
+        });
```
Comment (lolcode8) [inline src/containers/post/post.component.jsx:66]: This block of code should be within the `then` block of the previous part. Or you can use the async await syntax to make it all cleaner

File: src/containers/create-post/create-post.component.jsx [hunk 2]
```diff
@@ -4,23 +4,50 @@ import { withRouter } from "react-router-dom";
...
+    } else {
+      let imgPost = false;
+      const picID = parseInt(Math.random() * 1000000000000).toString();
+
+      if (imageOption1 && imageOption2) {
+        storage.ref(`/posts/${picID}-01`).put(imageOption1);
+        storage.ref(`/posts/${picID}-02`).put(imageOption2);
+        imgPost = true;
+      }
+      addPost({
```
Comment (lolcode8) [inline src/containers/create-post/create-post.component.jsx:37]: await this

File: src/containers/create-post/create-post.component.jsx [hunk 3]
```diff
@@ -4,23 +4,50 @@ import { withRouter } from "react-router-dom";
...
+  const [imageOption1, setImageOption1] = useState("");
+  const [imageOption2, setImageOption2] = useState("");
+  const [pictureUpload, setPictureUpload] = useState(false);
   const showSidebar = useSelector((state) => state.sidebarView.sidebarView);
   const user = useSelector((state) => state.users.user);
   const userID = user.userId;
   const username = user.username;
-  const avatarLink = user.avatar_link;
 
   const addPostToDB = () => {
```
Comment (lolcode8) [inline src/containers/create-post/create-post.component.jsx:21]: Make this function async so you can use await inside

File: src/containers/create-post/create-post.component.jsx [hunk 4]
```diff
@@ -4,23 +4,50 @@ import { withRouter } from "react-router-dom";
...
-    setOption2("");
-    history.push("/");
+    if (pictureUpload && (!imageOption1 || !imageOption2)) {
+      alert("Please upload 2 images");
+    } else if (!pictureUpload && !postText) {
+      alert("Please complete post with text");
+    } else if (!option1 || !option2) {
+      alert("Please fill in options");
+    } else {
+      let imgPost = false;
```
Comment (lolcode8) [inline src/containers/create-post/create-post.component.jsx:29]: Not a huge deal but better if you can avoid using let like we discussed earlier.

File: src/containers/create-post/create-post.component.jsx (hunk 3 above)
Comment (noahnovickf) [reply src/containers/create-post/create-post.component.jsx:21]: ??

File: src/containers/create-post/create-post.component.jsx (hunk 4 above)
Comment (noahnovickf) [reply src/containers/create-post/create-post.component.jsx:29]: I tried.

**Pull request merged**